src/Symfony/Bridge/PhpUnit/CoverageListener.php

```diff
@@ -15,11 +15,14 @@
 use PHPUnit\Framework\Test;
 use Symfony\Bridge\PhpUnit\Legacy\CoverageListenerTrait;
 
-if (class_exists('PHPUnit_Runner_Version') && version_compare(\PHPUnit_Runner_Version::id(), '6.0.0', '<')) {
-    class_alias('Symfony\Bridge\PhpUnit\Legacy\CoverageListener', 'Symfony\Bridge\PhpUnit\CoverageListener');
 // Using an early return instead of a else does not work when using the PHPUnit
 // phar due to some weird PHP behavior (the class gets defined without executing
 // the code before it and so the definition is not properly conditional)
+if (class_exists('PHPUnit_Runner_Version') && version_compare(\PHPUnit_Runner_Version::id(), '6.0.0', '<')) {
+    class_alias('Symfony\Bridge\PhpUnit\Legacy\CoverageListener', 'Symfony\Bridge\PhpUnit\CoverageListener');
+} elseif (version_compare(\PHPUnit\Runner\Version::id(), '7.0.0', '>=')) {
+    require_once __DIR__.'/CoverageListenerWithReturnTypes.php';
```

Comment:
I don't get why I have to do that :/
Without this line, I got 
```
PHP Warning:  Class 'Symfony\Bridge\PhpUnit\CoverageListenerWithReturnTypes' not found in /home/gregoire/dev/github.com/lyrixx/symfony/src/Symfony/Bridge/PhpUnit/CoverageListener.php on line 25
```

How to reproduce: 
* install phpunit 7+
* get my branch
* `cd src/Symfony/Bridge/PhpUnit/`
* `composer install`
* `phpunit`

Reply:
Strange. Related: in #26139 I'm proposing to completely split implems in separate classes, and do only aliases here. I'd suggest to do the same here.

Reply:
okay, I will mimic your PR